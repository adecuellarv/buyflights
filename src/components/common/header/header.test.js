import React from 'react';
import { Link } from 'react-router';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import Header from './header';

test('Header Test', () => {
    const component = renderer.create(
        <Router>
            <Header />
        </Router>
        ,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
    
    const testInstance = component.root;
    const allLinks = testInstance.findAllByType('a');
    expect(allLinks.length).toBe(2)
    
});