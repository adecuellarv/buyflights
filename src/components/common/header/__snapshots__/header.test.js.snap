// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Test 1`] = `
<div
  className="header"
>
  <div
    className="div-home-hd"
  >
    <a
      href="/"
      onClick={[Function]}
    >
      Inicio
    </a>
  </div>
  <a
    href="/reservas"
    onClick={[Function]}
  >
    <div
      className="cart-div"
    >
      <svg
        aria-hidden="true"
        className="svg-inline--fa fa-shopping-bag fa-w-14 icon-cart-hd"
        data-icon="shopping-bag"
        data-prefix="fas"
        focusable="false"
        role="img"
        style={Object {}}
        viewBox="0 0 448 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M352 160v-32C352 57.42 294.579 0 224 0 153.42 0 96 57.42 96 128v32H0v272c0 44.183 35.817 80 80 80h288c44.183 0 80-35.817 80-80V160h-96zm-192-32c0-35.29 28.71-64 64-64s64 28.71 64 64v32H160v-32zm160 120c-13.255 0-24-10.745-24-24s10.745-24 24-24 24 10.745 24 24-10.745 24-24 24zm-192 0c-13.255 0-24-10.745-24-24s10.745-24 24-24 24 10.745 24 24-10.745 24-24 24z"
          fill="currentColor"
          style={Object {}}
        />
      </svg>
    </div>
  </a>
</div>
`;
